### PR TITLE
Core module cleanup - Phase 1

### PR DESCRIPTION
**Pull request by ibgreen**
# For #58

### Background

- core must be small
- Fix bundle size analysis
- Start identifying code that makes bundler add node.js polyfills (buffer, base64, browser,...) and eliminate the root causes.

### CODE REVIEW

File: test/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```
Comment (Pessimistress) [inline test/webpack.config.js]: Why are we adding this? All tests are using the root webpack.config.js now

File: scripts/collect-metrics.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
```
Comment (Pessimistress) [inline scripts/collect-metrics.sh]: Contribute to ocular instead?

**Comment by Pessimistress** [comment]
Try upgrade to the latest ocular-dev-tools to fix CI.

**Comment by ibgreen** [comment]
Most of this has been addressed in other PRs.

**Pull request closed without merging**